benchmarks/BM_interpolators.cpp

```diff
@@ -46,7 +47,8 @@ static void doInterpolation(absl::Span<const float> input, absl::Span<float> out
 {
     const float kOutToIn = static_cast<float>(input.size()) / output.size();
 
-    for (size_t iOut = 0; iOut < output.size(); ++iOut) {
+    for (size_t iOut = sfz::config::excessFileFrames;
+            iOut < output.size() - sfz::config::excessFileFrames; ++iOut) {
```

Comment:
The fix seems inadequate. The excess count is not supposed to apply to output indices.
Plus would it fix anything in case we'd be upsampling by some higher factor?

The better way seems to over-allocate the input buffer as part of `SetUp`.

Reply:
Sorry, corrected